.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "workbench.colorCustomizations": {
+    "activityBar.background": "#1A2548",
+    "titleBar.activeBackground": "#243465",
+    "titleBar.activeForeground": "#F8F9FD"
+  }
+}
```

Comment:
Can you remove this file? Probably makes sense to add `.vscode` to the root `.gitignore`

Reply:
Yeah missed that.. sorry removing 🆗